### PR TITLE
PREAPPS-7867: Implementation of Truncated folder names

### PR DESCRIPTION
**Pull request by sandesh-telus**
- adde variable '--spacing-sidebar-sub-folder' for sidebar subfolder spacing

PR for [zm-x-web](https://github.com/Zimbra/zm-x-web/pull/5064)

### CODE REVIEW

File: css-variables.css [hunk 1]
```diff
@@ -113,5 +113,6 @@
 	--row-height-narrow: 72px;
 	--row-height-wide: 27px;
 	--tag-secondary-font-color: #ffffff;
-	--popover-row-border-color: #ccc
+	--popover-row-border-color: #ccc;
+	--spacing-sidebar-sub-folder: 20px;
```
Comment (silentsakky) [inline css-variables.css:117]: we already have spacing related variables

Comment (sandesh-telus) [reply css-variables.css:117]: used --line-height-computed variable to apply spacing for the sidebar sub-folder.

**Comment by silentsakky** [comment]
@sandesh-telus is this PR required?

**Comment by sandesh-telus** [comment]
> @sandesh-telus is this PR required?

No, it is not required.

**Pull request closed without merging**